mlrun/feature_store/ingestion.py

```diff
@@ -88,8 +92,12 @@ def init_featureset_graph(
     targets = [get_target_driver(target, featureset) for target in targets]
     for chunk in chunks:
         event = MockEvent(body=chunk)
+        if featureset.spec.entities[0] and isinstance(event.body, pd.DataFrame):
+            event.body = event.body.set_index(featureset.spec.entities[0].name)
         data = server.run(event, get_body=True)
         if data is not None:
+            if featureset.spec.entities[0] in data:
+                data.drop(columns=[featureset.spec.entities[0]], inplace=True)
```

Comment:
what happened before this additions? It doesn't connect with anything you wrote in the PR description.